site/develop/committers.md

```diff
@@ -54,6 +54,7 @@ Ferdinand Xu            | xuf          | Committer
 Yongzhi Chen            | ychena       | Committer
 Yiqun Zhang             | yqzhang      | Committer
 Aihua Xu                | zihuaxu      | Committer
+Quanlong Huang          | stigahuang   | Committer
```

Comment:
Oh, the list is `alphabetic` order by ID.

Reply:
Could you move your ID to a new location after `sseth`?

Reply:
Oops! Thanks for catching this!